### PR TITLE
Fix schema areas

### PR DESCRIPTION
**Pull request by dibericky**
- increase expanded level to 10
- added copy button in schema area
- fixed "shared collapse state" between example and request

#### Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [x] tests are included
- [ ] the documentation is updated or integrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: packages/api-explorer/src/components/JsonViewer/index.jsx [hunk 1]
```diff
@@ -6,25 +6,34 @@ import PropTypes from 'prop-types'
...
-  right: 10
+  right: 10,
+  display: 'grid',
+  gridAutoFlow: 'column',
+  gridGap: 10
 }
 
+const COLLAPSED_LEVEL = 1
+const EXPANDED_LEVEL = 10
+
```
Comment (davidebianchi) [inline packages/api-explorer/src/components/JsonViewer/index.jsx:22]:
```suggestion
```

File: packages/api-explorer/src/components/JsonViewer/index.jsx [hunk 2]
```diff
@@ -6,25 +6,34 @@ import PropTypes from 'prop-types'
...
+
 export default function JsonViewer({ schema, missingMessage }) {
   const [isCollapsed, setIsCollapsed] = useState(true)
 
   if (!schema) {
     return  <FormattedMessage id={missingMessage} defaultValue={'missing schema'} />
   }
 
+
```
Comment (davidebianchi) [inline packages/api-explorer/src/components/JsonViewer/index.jsx:31]:
```suggestion
```

File: packages/api-explorer/src/components/Response/index.jsx [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
+import CopyCode from '../CopyCode'
 
 const ResponseMetadata = require('./ResponseMetadata');
 const ResponseBody = require('./ResponseBody');
 const showCodeResults = require('../../lib/show-code-results');
 const contentTypeIsJson = require('../../lib/content-type-is-json');
 
 const Oas = require('../../lib/Oas');
-const CopyCode = require('../CopyCode');
+
```
Comment (davidebianchi) [inline packages/api-explorer/src/components/Response/index.jsx:15]:
```suggestion
```

File: packages/api-explorer/src/components/JsonViewer/CopyButton.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import {Button} from 'antd'
+import {FormattedMessage} from 'react-intl'
+
+import CopyText from '../CopyText'
+
+function stringify (schema) {
+    try {
+      const stringified = JSON.stringify(schema)
+      return stringified
+    }catch(err) {
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/components/JsonViewer/CopyButton.jsx:12]:
```suggestion
    } catch(err) {
```

File: packages/api-explorer/src/components/JsonViewer/CopyButton.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {Button} from 'antd'
+import {FormattedMessage} from 'react-intl'
+
+import CopyText from '../CopyText'
+
+function stringify (schema) {
+    try {
+      const stringified = JSON.stringify(schema)
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/components/JsonViewer/CopyButton.jsx:10]: Why not directly returning  JSON.stringify result?

File: packages/api-explorer/src/CodeSample.jsx [hunk 6]
```diff
@@ -2,16 +2,16 @@ import React from 'react'
 import {FormattedMessage} from 'react-intl';
 
 import BlockWithTab from './components/BlockWithTab'
+import CopyCode from './components/CopyCode'
 import colors from './colors';
 
+
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/CodeSample.jsx:8]:
```suggestion
```

File: packages/api-explorer/src/components/SchemaTabs/index.jsx [hunk 7]
```diff
@@ -99,7 +99,13 @@ export default class SchemaTabs extends Component {
...
         example = jsf.generate(schema)
       }
-      return <JsonViewer missingMessage={'schemaTabs.missing.example'} schema={example} />
+      return (
+        <JsonViewer 
+          missingMessage={'schemaTabs.missing.example'} 
+          schema={example} 
+          key={'json-viewer-example'}
+        />
+        )
```
Comment (fredmaggiowski) [inline packages/api-explorer/src/components/SchemaTabs/index.jsx:108]:
```suggestion
      )
```

**Pull request merged**